README.md

```diff
@@ -1,13 +1,13 @@
 ## gorilla/feeds
 [![GoDoc](https://godoc.org/github.com/gorilla/feeds?status.svg)](https://godoc.org/github.com/gorilla/feeds) [![Build Status](https://travis-ci.org/gorilla/feeds.png?branch=master)](https://travis-ci.org/gorilla/feeds)
 
-feeds is a web feed generator library for generating RSS and Atom feeds from Go
+feeds is a web feed generator library for generating RSS, Atom and JSON feeds from Go
 applications.
 
 ### Goals
 
  * Provide a simple interface to create both Atom & RSS 2.0 feeds
- * Full support for Atom and RSS2.0 spec elements
+ * Full support for Atom, RSS2.0, and JSON Feed Version 1 spec elements
```

Comment:
Have the `JSON Feed` text link to https://jsonfeed.org/ - being a very new spec, package users are less likely to be familiar with it.